### ISSUE TITLE
Any improvements welcome!

### ISSUE THREAD

**Issue by connorphee** [connorphee/league_app#4]
This is a very raw and newer project. I am open to any ideas or directions you suggest! Feel free to comment here or open a new issue if you have something you would like to add!

**Comment by AK-007** [comment]
Adding a Readme.md would be great

**Comment by connorphee** [comment]
Yes I absolutely need to do this. I will add a readme and contributing doc ASAP in the morning!

**Comment by arianahl** [comment]
The app currently requires an email but it never checks if the email is valid. Most of the apps out there would send an email verification and then create the user once the verification link is clicked by the user. 
Would you like to add this feature @connorphee? I'm happy to work on it if you assign a new issue to me 😄

**Comment by connorphee** [comment]
@arianahl checkout out #24
